.changes/unreleased/Fixes-20230912-133327.yaml

```diff
@@ -1,5 +1,5 @@
 kind: Fixes
```

Comment:
We're not considering this a bug since we're not backporting it. We're thinking of it as a new feature (the ability to swap MVs with table/view and vice versa). The body would also probably change given that context.